src/mngt/ocf_mngt_cache.c

```diff
@@ -2097,11 +2110,7 @@ void ocf_mngt_cache_stop(ocf_cache_t cache,
 
 	ENV_BUG_ON(!cache->mngt_queue);
 
-	result = ocf_pipeline_create(&pipeline, cache,
-			&ocf_mngt_cache_stop_pipeline_properties);
-	if (result)
-		OCF_CMPL_RET(cache, priv, -OCF_ERR_NO_MEM);
-
+	pipeline = cache->stop_pipeline;
 	context = ocf_pipeline_get_priv(pipeline);
 
 	context->cmpl = cmpl;
```

Comment:
Please replace error handling in line 2122 (env_strncpy) with BUG_ON - at this point we don't expect to receive any kind of error since all success conditions verified in advance. This would make it clear that stop does not return error code other than cache write error.